John conroy/storybook (#1915)
hubmapconsortium/portal-ui: README.md

<details><summary>For React</summary>

- Components with tests or styles should be placed in to their own directory.
- Styles should follow the `style.*` pattern where the extension is js for styled components or css for stylesheets.

> - Tests and stories should follow the `*.spec.js` and `*.stories.js` patterns respectively where the prefix is the name of the component.

Reviewer: ```suggestion
- Tests should follow the `*.spec.js` pattern...
- and stories should follow the `*.stories.js` pattern. For both, the prefix is the name of the component.
```

npm run lint:fix
```

### Storybook

> To start storybook locally run `npm run storybook`.

Reviewer: Would it be useful to just wrap this into `dev-start.sh`?

Author: Added to `dev-start.sh`.

npm run lint:fix
```

### Storybook

> To start storybook locally you can run `npm run storybook` or visit [localhost:6006](http://localhost:6006). after you have run `./dev-start.sh`.

Reviewer: ```suggestion
To start storybook locally you can either run `./dev-start.sh`, or just `npm run storybook`,
and after it has started, visit [localhost:6006](http://localhost:6006).
```
